src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1286,24 +1286,26 @@ If the task has not *IN_ERROR* status, the _Mark as finished and resume_ option
 
 Add a "NOTIFICATION_EVENTS" Generic information in the workflow level, to list events on which a user wants to be notified. List is comma-separated and should be taken from(events are not case sensitive):
 
-* *Job change priority*
-* *Job In-Error*
-* *Job paused*
-* *Job pending to finished*
-* *Job pending to running*
-* *Job restarted from error*
-* *Job resumed*
-* *Job running to finished*
-* *Job submitted*
+* *Job change priority (or JOB_CHANGE_PRIORITY)*
```

Comment:
I would suggest:
* `JOB_CHANGE_PRIORITY`: Job change of priority

The same for all below

Reply:
then the users will be confused what exactly should be inserted as generic info.